contents/handbook/product/ai-playbook.md

```diff
@@ -0,0 +1,71 @@
+---
+title: In-product AI playbook
+sidebar: Handbook
+showTitle: true
+---
+
+Throughout the 2020s, LLM-enabled products will eat those which stay in the past – just see Cursor vs. JetBrains. LLM-based PostHog features already are advancing [our mission](/handbook/why-does-posthog-exist), but this journey has just begun. New use cases are becoming viable as models improve in reliability and cost, and YC _will_ fund every AI-oriented founder with the ambition to overtake us. We can't be any less ambitious.
+
+This doc describes how PostHog goes on to thrive, instead of being eaten. You're not looking at a high-level exec strategy. You're looking at the **in-product AI playbook** for every team at PostHog.
+
+There are two angles to our in-product AI:
+
+## 1. PostHog Max – the world's first artificial product manager
+
+Product-minded developers want to build the right thing. They already can _build_, and AI dev tooling – like Cursor's agent – helps build faster.
+
+PostHog's agent helps build _the right thing_. Named Max, this is our core AI product. It's [deep research](https://openai.com/index/introducing-deep-research/) for product questions, with knowledge of data and tools living in PostHog.
+
+Max is owned by the [AI Product Manager team](/teams/ai-product-manager) – see its page for this product's roadmap.
+
+## 2. PostHog Intelligence - acceleration throughout
+
+Max, the agentic assistant over chat, is not the be-all and end-all. AI features can be found embedded throughout PostHog products in targeted ways. We call this PostHog Intelligence.
+
+The principles behind PostHog Intelligence are:
+
+### Everyone can build AI features
+
+While the AI PM team owns Max as the one LLM-powered _product_, **every product team has the mandate to solve problems with AI**.
+
+As you build, just keep the AI PM team in the loop, so that they can offer expertise and later integrate your work into Max. (Max becomes more capable with every new skill we give it.)
+
+### We use LLMs to solve real problems
+
+Use AI features to solve actual customer problems, or problems we experience ourselves. The shiny tech brings a little more temptation to build toy demos – keep that energy, but focus it on solving concrete customer needs.
+
+LLMs are great for a few types of needs:
```

Comment:
every opportunity to plug the phrase "Calculator for words" i will take

https://simonwillison.net/2023/Apr/2/calculator-for-words/

:p